flaskr/__init__.py

```diff
@@ -0,0 +1,20 @@
+from flask import Flask, request, render_template
+from src import arbitrage
+
+
+def create_app(test_config=None):
+    # flask server
+    app = Flask(__name__, instance_relative_config=True)
+
+    @app.route('/', methods=("GET", "POST"))
+    def hello_world():
```

Comment:
Look up REST api architecture.

POST is used to create a resource and GET is used to get it.

There should be no reason you're POSTing '/'.

A better architecture is to take a GET request for the ticker i.e. '/aapl' and return the result for Apple stock.